server/lib/src/be/idl_sqlite.rs

```diff
@@ -1712,7 +1707,7 @@ impl IdlSqliteWriteTransaction {
 
 impl IdlSqlite {
     pub fn new(cfg: &BackendConfig, vacuum: bool) -> Result<Self, OperationError> {
-        if cfg.path.is_empty() {
+        if cfg.path == Path::new("") {
```

Comment:
how is that better than `is_empty()`?

edit:
Likely a PathBuf in which case `.as_os_str().is_empty()` should probably be used instead (at least that looks to be the standard way of doing this)

Reply:
Because there isn't .is_empty() on Path as you found out :)